spec/support/helpers/login_helpers.rb

```diff
@@ -3,6 +3,18 @@ def login(member)
     allow_any_instance_of(ApplicationController).to receive(:current_user).and_return(member)
   end
 
+  def login_mock_omniauth(member, login_link = 'Sign in')
+    OmniAuth.config.mock_auth[:github] = {
+      provider: member.auth_services.first.provider,
+      uid: member.auth_services.first.uid,
+      credentials: {
+        token: 'mock_token',
+        secret: 'mock_secret'
+      }
+    }
+    click_on login_link
+  end
+
```

Comment:
The original login method wasn't working for the new event tests so I created this 'login_mock_omniauth" method. It actually logs in the member with mock credentials.